Src/Newtonsoft.Json.UnityConverters/EnumerableVectorConverter.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using UnityEngine;
+#if (PORTABLE)
+using System.Reflection;
+#endif
+
+namespace Newtonsoft.Json.UnityConverters
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class EnumerableVectorConverter<T>: JsonConverter
+    {
+        private static readonly VectorConverter VectorConverter = new VectorConverter();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A static field in a generic type is not shared among instances of different close constructed types.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4846771)